exercises/grains/GrainsTest.swift

```diff
@@ -3,51 +3,90 @@ import XCTest
 
 
 
-// Return multiple types and compare by overloading ==
 
 class GrainsTest: XCTestCase {
     
     func testInvalidInput1() {
-        XCTAssertTrue((result: 0, error: "Input[65] invalid. Input should be between 1 and 64 (inclusive)") == Grains.square(65))
+        var throwsInputTooHighError = false
+        
+        defer {
+            XCTAssertTrue(throwsInputTooHighError)
+        }
+        
+        do {
+            let _ = try Grains.square(65)
+        } catch Grains.Error.InputTooHigh(let message) {
```

Comment:
Unfortunately, it's not able to infer the error types that way: it only works if I fully qualify the name.